clusters/k8s-infra-dev-cluster-turnup/10-clusterconfiguration.tf

```diff
@@ -120,14 +143,24 @@ resource "google_container_cluster" "cluster" {
 
   // Configure cluster addons
   addons_config {
+    horizontal_pod_autoscaling {
+      disabled = false
+    }
+    http_load_balancing {
+      disabled = false
+    }
     network_policy_config {
       disabled = false
     }
   }
 
-  // GKE clusters are critical objects and should not be destroyed
-  // IMPORTANT: should be true on production cluster
-  lifecycle {
-    prevent_destroy = false
+  // Enable PodSecurityPolicy enforcement
+  pod_security_policy_config {
```

Comment:
Doesn't turning on PSP enforcement require that we already have all applications set up with a PSP?

Reply:
Good point.  I missed the big red warning. :)  Will update.